sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperation.cs

```diff
@@ -113,12 +113,12 @@ public void AnalyzeOperation()
                 docNumber = 1;
                 foreach (RecognizePiiEntitiesActionResult piiActionResults in piiActionsResults)
                 {
-                    foreach (RecognizePiiEntitiesResult result in piiActionResults.Result)
+                    foreach (RecognizePiiEntitiesResult documentsResults in piiActionResults.DocumentsResults)
```

Comment:
Same here and in other applicable foreach blocks.

Reply:
By the way, in the other loop we had a property called `DocumentResults`, but in this one it's `DocumentsResults`. Is this difference (document/documents) intentional?

Reply:
plop great eyes! they should all be `DocumentSResults` both plural.
Will update